src/lib/types.ts

```diff
@@ -443,6 +444,8 @@ export type SignInWithOAuthCredentials = {
     queryParams?: { [key: string]: string }
     /** If set to true does not immediately redirect the current browser context to visit the OAuth authorization page for the provider. */
     skipBrowserRedirect?: boolean
+    /** If set to 'pkce' PKCE flow. Defaults to the 'implicit' flow otherwise */
+    flowType?: OAuthFlowType
```

Comment:
doesn't seem clear that it defaults to `implicit` because it's only set as a default in `_handleProviderSignIn()` so this might be abit confusing for someone looking through the code for the first time - would be slightly clearer with the suggestion above